workflows/sesv2_weekly_mailer/SPECIFICATION.md

```diff
@@ -0,0 +1,78 @@
+# SES v2 Coupon Newsletter Workflow Specification
+
+Use the Amazon Simple Email Service (SES) v2 API to manage a subscription list for a weekly newsletter.
+
+## Prepare the Application
+
+1. Create an email identity.
+
+   - Retrieve the verified email address from the `VERIFIED_EMAIL_ADDRESS` environment variable.
+   - Operation: **CreateEmailIdentity**
+     - `EmailIdentity`: Value of the `VERIFIED_EMAIL_ADDRESS` environment variable.
+
+2. Create a contact list with the name `weekly-coupons-newsletter`.
+   - Operation: **CreateContactList**
+     - `ContactListName`: `weekly-coupons-newsletter`
+
+## Gather Subscriber Email Addresses
+
+1. Prompt the user to enter 3 to 5 email addresses for subscribing to the newsletter.
```

Comment:
This seems like too many. I wouldn't want to enter 3+ email addresses into an example.